lib/membrane/rtp/session_bin.ex

```diff
@@ -52,6 +52,15 @@ defmodule Membrane.RTP.SessionBin do
   @type extension_t ::
           {extension_name :: atom(), extension_config :: Membrane.ParentSpec.child_spec_t()}
 
+  @typedoc """
+  A tuple with a name and element's module/struct that will be responsible for arbitrary packet filtering
+  inside `Membrane.RTP.StreamReceiveBin`.
+
+  A filter can be responsible e.g. for dropping silent audio packets when encountered VAD extension data in the
+  packet header.
+  """
+  @type packet_filter_t :: {name :: atom(), element :: module() | struct()}
```

Comment:
```suggestion
  @type packet_filter_t :: {Membrane.Child.name_t(), Membrane.ParentSpec.child_spec_t()}
```

Reply:
and mention that each filter should have one input and output pad, named accordingly